scenarios/27_Orannon.cfg

```diff
@@ -912,7 +1123,13 @@
         {MSGm_Lorin _"FOUL mage. You should have said: 'I am a FOUL mage'."}
         [objectives]
             side=1
-            note=_"In this scenario, only Gawen and Lorin may interrogate enemy leaders"
+#ifdef NIGHTMARE
+            #po: no hints for NIGHTMARE difficulty:
+            note=_"You will have to figure out who has interrogations in this scenario on your own."
```

Comment:
Same, I don't think a note is needed.

Reply:
Removed in [086c1d2](https://github.com/nemaara/A_New_Order/pull/53/commits/086c1d25c13ea2fc5549ae746b68eb2476cc283a).